src/main.js

```diff
@@ -54,7 +54,8 @@ module.exports = {
           // Do not try to fix if linting should be disabled
           const fileDir = Path.dirname(filePath)
           const configPath = getConfigPath(fileDir)
-          if (configPath === null && disableWhenNoEslintConfig) return
+          const noProjectConfig = (configPath === null || isConfigAtHomeRoot(configPath))
+          if (noProjectConfig && disableWhenNoEslintConfig) return
```

Comment:
I'm personally not a fan of one line `if` statements, but I'll leave that up to you.